tests/test_flow/test_relative_cache_path.py

```diff
@@ -1,21 +1,15 @@
 import os
-import pytest
 import shutil
 
 
-@pytest.fixture
-def builder(builder):
```

Comment:
This test fails if we have a separate builder fixture. Moving it inside the test fixed the problem.